app/apptesting/test_common.go

```diff
@@ -0,0 +1,56 @@
+package apptesting
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	xauthsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+// CreateTestTx is a helper function to create a tx given multiple inputs.
+func CreateTestTx(privs []cryptotypes.PrivKey, accNums []uint64, accSeqs []uint64, chainID string, clientCtx client.Context, txBuilder client.TxBuilder) (xauthsigning.Tx, error) {
```

Comment:
We should place all context parameters related at the beginning of function

Reply:
@dao-notional done refactor sir